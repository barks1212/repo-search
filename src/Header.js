import React from 'react';

const Header = () => (
  <section className="hero has-text-centered is-info" id="header">
    <section className="hero-body">
      <section className="container">
        <h1 className="title">
          <i className="fab fa-github-alt"></i>
          Github Repo Search
          <i className="fab fa-github-alt"></i>
        </h1>
      </section>
    </section>
  </section>
);

export default Header;