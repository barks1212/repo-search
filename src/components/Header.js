import React from 'react';

const Header = () => (
  <section className="hero has-text-centered is-info is-mobile" id="header">
    <section className="hero-body">
      <section className="container">
        <h1 className="title is-size-4-mobile">
          <i className="fab fa-github-alt"></i>
          Github Repo Search
          <i className="fab fa-github-alt"></i>
        </h1>
      </section>
    </section>
  </section>
);

export default Header;