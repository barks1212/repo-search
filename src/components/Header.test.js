import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('<Header>', () => {
  it('renders as expected without crashing and matches snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});