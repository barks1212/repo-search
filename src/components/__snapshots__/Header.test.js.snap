// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders as expected without crashing and matches snapshot 1`] = `
<section
  className="hero has-text-centered is-info is-mobile"
  id="header"
>
  <section
    className="hero-body"
  >
    <section
      className="container"
    >
      <h1
        className="title is-size-4-mobile"
      >
        <i
          className="fab fa-github-alt"
        />
        Github Repo Search
        <i
          className="fab fa-github-alt"
        />
      </h1>
    </section>
  </section>
</section>
`;
